pkg/artifactory/resource_artifactory_backup.go

```diff
@@ -0,0 +1,208 @@
+package artifactory
+
+import (
+	"context"
+	"github.com/go-resty/resty/v2"
```

Comment:
Please group the import statements per convention

Reply:
done